### PR TITLE
Add create function to the default permissions for transformer RWC

### PR DESCRIPTION
**Pull request by ram-kishore-odi**
Please see the plan log below for reference- 

****************************************************************************
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

module.elt.module.analytics.snowflake_grant_privileges_to_account_role.public["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "public" {
        id                = "\"ANALYTICS_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnSchema|\"ANALYTICS_DEV\".\"PUBLIC\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

module.elt.module.analytics.snowflake_grant_privileges_to_account_role.schemas["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "schemas" {
        id                = "\"ANALYTICS_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnFutureSchemasInDatabase|\"ANALYTICS_DEV\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

 module.elt.module.raw.snowflake_grant_privileges_to_account_role.public["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "public" {
        id                = "\"RAW_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnSchema|\"RAW_DEV\".\"PUBLIC\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

module.elt.module.raw.snowflake_grant_privileges_to_account_role.schemas["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "schemas" {
        id                = "\"RAW_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnFutureSchemasInDatabase|\"RAW_DEV\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

module.elt.module.transform.snowflake_grant_privileges_to_account_role.public["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "public" {
        id                = "\"TRANSFORM_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnSchema|\"TRANSFORM_DEV\".\"PUBLIC\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

module.elt.module.transform.snowflake_grant_privileges_to_account_role.schemas["READWRITECONTROL"] will be updated in-place
  ~ resource "snowflake_grant_privileges_to_account_role" "schemas" {
        id                = "\"TRANSFORM_DEV_READWRITECONTROL\"|false|false|CREATE PROCEDURE,CREATE FILE FORMAT,CREATE TABLE,CREATE VIEW,MODIFY,CREATE STAGE,CREATE PIPE,CREATE TEMPORARY TABLE,USAGE,CREATE MATERIALIZED VIEW,MONITOR|OnSchema|OnFutureSchemasInDatabase|\"TRANSFORM_DEV\""
      ~ privileges        = [
          + "CREATE FUNCTION",
            # (11 unchanged elements hidden)
        ]
        # (5 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 6 to change, 0 to destroy.

****************************************************************************

### CODE REVIEW

**Comment by ian-r-rose** [comment]
@ram-kishore-odi when you create a pull request or issue, can you fill out the project management metadata in the sidebar?

File: terraform/snowflake/modules/database/main.tf [hunk 1]
```diff
@@ -51,6 +51,7 @@ locals {
       "CREATE MATERIALIZED VIEW",
       "CREATE PIPE",
       "CREATE PROCEDURE",
+      "CREATE SCHEMA",
```
[the remark this answers is not in the document]
Comment (ram-kishore-odi) [reply terraform/snowflake/modules/database/main.tf]: Sorry about it. It was accidental. I even updated the title of the story 4 days back to _add create function_ in the title here - https://github.com/orgs/cagov/projects/31/views/39?sliceBy%5Bvalue%5D=ram-kishore-odi&pane=issue&itemId=90872592&issue=cagov%7Cdata-infrastructure%7C449 but typed Schema while making this one line change.  I fixed it and also updated the terraform plan log output above. Please review it.

**Comment by ram-kishore-odi** [comment]
> @ram-kishore-odi when you create a pull request or issue, can you fill out the project management metadata in the sidebar?

I updated the project management metadata in the sidebar. Please let me know if additional changes are needed

**Review by ian-r-rose — APPROVED** [review]
Looks good to me, thanks @ram-kishore-odi!

**Pull request merged**